### PR TITLE
Fix amount send to mm and feeFactor usage

### PR DESCRIPTION
**Pull request by Xaber20110202**
1. fix amount send to mm when side is `BUY` situation. The bug maybe like this, when `side` is `BUY`, amount is `0.333` to mm, but the actual executed amount maybe is `0.334`
2. fix that feeFactor got from the takerToken of the order, but the actual feeFactor should got from the makerToken of the order (Although the feeFactor configuration of all tokens is the same now and before)
3. remove websocket url config field

### CODE REVIEW

File: src/utils/token.ts [hunk 1]
```diff
@@ -56,19 +58,55 @@ export const getTokenBySymbol = (tokens, symbol) => {
...
+    const { amount, side } = query
+
+    if (side === 'BUY') {
+      const config = updaterStack.markerMakerConfigUpdater.cacheResult
+      const tokenConfigs = updaterStack.tokenConfigsFromImtokenUpdater.cacheResult
+      // 注意：query 上，后端传递的是 feefactor，而不是 feeFactor
+      // 但是，Token Config 返回的配置是 feeFactor
+      const queryFeeFactor = query.feefactor
+      // 用户 BUY base, 手续费就是 base 的 Token，即 order的 makerToken —— 对应做市商转出的币，用户收到的币
+      const foundTokenConfig = tokenConfigs.find(t => t.symbol === query.base)
```
Comment (kaichen) [inline src/utils/token.ts:83]: maybe compare upper case string here?

Comment (Xaber20110202) [reply src/utils/token.ts]: > maybe compare upper case string here?

🙏  已处理

File: src/utils/token.ts [hunk 2]
```diff
@@ -56,19 +58,55 @@ export const getTokenBySymbol = (tokens, symbol) => {
...
+  }).forEach(([key, value]) => {
+    if (value) {
+      Object.assign(result, {
+        [key]: value,
+      })
+    }
+  })
+
+  return result
 }
```
Comment (kaichen) [inline src/utils/token.ts:112]: here mixes two functions, symbol transform, and amount process.

And this `token.ts` file should only force on the token property, and `rate.ts` file should contain all amount processing

Comment (Xaber20110202) [reply src/utils/token.ts:57]: this function has been moved to `helper.ts`

**Pull request merged**